js/data.js

```diff
@@ -1210,5 +1419,16 @@ var data = {
         cp: 6,
         dozent: ["Prof. Dr. Robert Hirschfeld", "Tobias Pape", "Tim Felgentreff", "Bastian Steinert"],
         nameLV: "Virtuelle Maschinen und Ausführungsumgebungen"
+    },
+    visualSoftwareAnalytics: {
+        kurz: "Visual Software<br />Analytics",
+        kennung: [ITSE, HCT, SAMT],
+        lehrform: ["Seminar"],
+        modul: ["IT-Systems Engineering", "Vertiefungsgebiete"],
+        semester: [ss15],
+        benotet: true,
+        cp: 6,
+        dozent: ["Prof. Dr. Jürgen Döllner", "Willy Scheibel", "Sebastian Hahn"],
+        nameLV: "Visual Software Analytics"
```

Comment:
There is a seminar SoftwareAnalytics but it is supposedly part of the winter term? I think since the names of the lecture differ it should be possible to attend both lectures, which means onehundredandtwenty should offer users to take both.